.jenkins/pytorch/build.sh

```diff
@@ -16,20 +16,6 @@ fi
 
 source "$(dirname "${BASH_SOURCE[0]}")/common.sh"
 
-# For distributed, four environmental configs:
-# (1) build with only NCCL
-# (2) build with NCCL and MPI
-# (3) build with only MPI
-# (4) build with neither
-if [[ "$BUILD_ENVIRONMENT" == *-xenial-cuda9*gcc7* ]] || [[ "$BUILD_ENVIRONMENT" == *-xenial-cuda9*gcc5* ]] || [[ "$BUILD_ENVIRONMENT" == *-xenial-cuda10.1-* ]] || [[ "$BUILD_ENVIRONMENT" == *-trusty-py2.7.9* ]]; then
-  if [[ "$BUILD_ENVIRONMENT" == *-trusty-py2.7.9* ]]; then
-    # TODO: move this to Docker
-    sudo apt-get -qq update
-    sudo apt-get -qq install openmpi-bin libopenmpi-dev
-  fi
-  sudo mkdir -p /var/run/sshd
```

Comment:
I am not quite sure we still need to make this directory for those cases, though I suspect we don't.

Reply:
I've decided from looking at test signal that we don't need to make such `/var/run/sshd` dir.